package.json

```diff
@@ -453,6 +453,34 @@
             "default": null,
             "markdownDescription": "Font family of Runme notebook terminal"
           },
+          "runme.terminal.cursorStyle": {
+            "type": "string",
+            "enum": [
+              "block",
+              "bar",
+              "underline"
+            ],
+            "default": "bar",
+            "markdownDescription": "The style of the Runme notebook terminal cursor."
+          },
+          "runme.terminal.cursorBlink": {
+            "type": "boolean",
+            "default": true,
+            "markdownDescription": "Defines whether the Runme notebook terminal cursor blinks."
+          },
+          "runme.terminal.cursorWidth": {
+            "type": "number",
+            "markdownDescription": "The width of the cursor in CSS pixels when `cursorStyle` is set to 'bar'."
+          },
+          "runme.terminal.smoothScrollDuration": {
+            "type": "number",
+            "default": 0,
+            "markdownDescription": "The duration to smoothly scroll between the origin and the target in milliseconds. Set to 0 to disable smooth scrolling and scroll instantly."
+          },
+          "runme.terminal.scrollback": {
```

Comment:
I think is better to have a default value of 100, see integrated terminal settings for VS Code

Reply:
Currently the implemented behavior mimics what we have in `main` which is not setting the value at all and have it default to the [default value](https://github.com/xtermjs/xterm.js/blob/bdb25c2dbee65af8afa36dca816af51fe5ab1d5d/src/common/services/OptionsService.ts#L33). Are you suggesting to explicitly set the default value to `100`?